### PR TITLE
Add IoT sample for MIMXRT1060

### PR DESCRIPTION
**Pull request by hihigupt**

### CODE REVIEW

File: demo/Common/mimxrt1060/main.c [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+/* MDIO operations. */
+#define mainMDIO_OPS                        enet_ops
+
+/* PHY operations. */
+#define mainPHY_OPS        		            phyksz8081_ops
+
+/* ENET clock frequency. */
+#define mainCLOCK_FREQ                      CLOCK_GetFreq( kCLOCK_IpgClk )
+
+
```
Comment (danewalton) [inline demo/Common/mimxrt1060/main.c:113]: nit nit: uncrustify this at all?

**Review by danewalton — APPROVED** [review]
Just to clarify here....this still uses the same task and demo that the other samples use correct? Because I see no calls to any middleware stuff here

File: demo/Common/mimxrt1060/main.c (hunk 1 above)
Comment (hihigupt) [reply demo/Common/mimxrt1060/main.c]: Updated

**Comment by hihigupt** [comment]
> Just to clarify here....this still uses the same task and demo that the other samples use correct? Because I see no calls to any middleware stuff here

Reason is because the FreeRTOS task remain same, I am just calling the function that is generic to all the ports.

vStartSimpleMQTTDemo

**Pull request merged**